roles/crawler/tasks/configure-databases.yml

```diff
@@ -119,3 +119,25 @@
     or False in schema_result.results|map(attribute='skipped')|list"
   become: true
   become_user: postgres
+
+# Populate the raw schema in the prod database with data
+- block:
+  - name: "See how many rows are in the raw schema."
+    command: psql -c 'select count(*) from raw.{{ item }}'
+    register: raw_schema_population_result
+    with_items:
+      - crawls
+      - hs_history
+      - frontpage_examples
+      - frontpage_traces
+    always_run: true
+    changed_when: false
+
+  - debug: var=raw_schema_population_result.results
```

Comment:
Do you still want these `debug` line in here? FYI recent versions of Ansible support the `verbosity` parameter on `debug` tasks, so the associated message will only display with e.g. `-vvv` if `verbosity: 3`. http://docs.ansible.com/debug_module.html